app/models/reporting/return.rb

```diff
@@ -26,8 +26,9 @@ def populate!
     end
 
     def source_data(ids)
```

Comment:
Rename `ids` to be `enrollment_ids` to make it clear what is passed?
Maybe rename to `service_data` to go with `enrollment_data`?